strapi/strapi
# feat(content-releases): Redirect to /content-releases/:releaseId onCreate and add header on details page

### What does it do?

Creates the Release details page and add the redirect to that page when you submit a new Release

### Why is it needed?

Because we need to redirect to the Details page of the new created release (for the moment the data are mocked)

### How to test it?

You can go to the Release page and create a new Release, when you submit the new release you will be redirected to the details page,
in the details page you can see
 - The release name as title
 - The quantity of entries on that release (We show “No entries” if there is no entries)
 - A right-side space for buttons
